model/src/gateway/presence/minimal_activity.rs

```diff
@@ -0,0 +1,31 @@
+use super::{Activity, ActivityType};
+
+#[derive(Clone, Debug, Eq, Hash, PartialEq)]
```

Comment:
This should implement serialize so it can be used to update presence over the gateway

Reply:
The `impl From<MinimalActivity> for Activity` covers this without breaking any other types or methods